src/editor/Renderer.cpp

```diff
@@ -504,22 +514,8 @@ void Renderer::renderLoop() {
 		glfwSwapBuffers(window);
 
 		if (reloading && fgdFuture.wait_for(chrono::milliseconds(0)) == future_status::ready) {
-			delete pointEntRenderer;
-			delete fgd;
-			
-			pointEntRenderer = (PointEntRenderer*)swapPointEntRenderer;
-			fgd = pointEntRenderer->fgd;
-
-			for (int i = 0; i < mapRenderers.size(); i++) {
-				mapRenderers[i]->pointEntRenderer = pointEntRenderer;
-				mapRenderers[i]->preRenderEnts();
-				if (reloadingGameDir) {
-					mapRenderers[i]->reloadTextures();
-				}
-			}
-
+			reloadFgds();

```

Comment:
Rename to `postLoadFgds()`. This block is entered after FGDs have loaded.

Reply:
ok